Remove average command (#26)
AY2122S1-CS2103-F10-1/tp: docs/UserGuide.md

Format: `add applicant n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS pos/POSITION​`

<div markdown="span" class="alert alert-primary">:bulb: **Tip:**

> A person can have any number of tags (including 0). <br>

Reviewer: ```suggestion
An applicant can have any number of tags (including 0). <br>
```
